src/Aspire.Dashboard/Components/Pages/Metrics.razor

```diff
@@ -2,7 +2,7 @@
 @page "/metrics/resource/{applicationName}"
 @page "/metrics/resource/{applicationName}/meter/{meterName}"
 @page "/metrics/resource/{applicationName}/meter/{meterName}/instrument/{instrumentName}"
-@page "/metrics/resource/{applicationName}/dashpages"
+@page "/metrics/resource/{applicationName}/highlights"
```

Comment:
It doesn't need to happen in this PR, but I think the highlight name/id should be in the URL.

That means there would there would be two routes for it:
```
@page "/metrics/resource/{applicationName}/highlights"
@page "/metrics/resource/{applicationName}/highlights/{highlightId}"
```

Reply:
I agree with you, I've moved it into the url